styles.css

```diff
@@ -0,0 +1,116 @@
+body {
+  font-family: 'Roboto', sans-serif;
+}
+table {
+  border-radius: 5px;
+  margin-left: 13%;
+  border: 2px solid;
+  border-color: #BC9437;
+  padding: 0;
+  border-spacing: 0;
+}
+table th {
+  width: 100px;
+  border: 1px solid;
+  font-family: 'Bungee', cursive;
+  border-color: #BC9437;
+}
+.frame {
+  border: 1px solid;
+  border-top: none;
+  width: 50px;
+  border-color: #BC9437;
+}
+.frameScore {
+  height: 45px;
+  border: 1px solid;
+  border-top: none;
+  border-left: none;
+  border-bottom: none;
+  border-color: #BC9437;
+}
+.scoreButton {
+  background-color: #F1BE47;
+  border: none;
+  font-size: 18px;
+  padding: 32px 16px;
+  border-radius: 10%;
+  box-shadow: 0 9px #999;
+  width: 80px;
+  font-family: 'Bungee Shade', cursive;
+}
+.scoreButton:active {
+  box-shadow: 0 5px #666;
+  transform: translateY(4px);
+}
+.scoreButton:focus {
+  outline: none;
+}
+article#error {
+  text-align: center;
+  font-size: 30px;
+  font-family: 'Bungee', cursive;
+  position: relative;
+  margin-top: 50px;
+}
+h1#title {
+  text-align: center;
+  font-family: 'Bungee Shade', cursive;
+  font-size: 60px;
+}
+h2#currentScore {
+  font-family: 'Bungee Shade', cursive;
+  margin-left: 25%;
+  font-size: 30px;
+  float: left;
+}
+h2#reset {
+  font-family: 'Bungee Shade', cursive;
+  margin-right: 25%;
+  font-size: 30px;
+  float: right;
+}
+h2#reset:hover {
+  color: white;
+}
+img#ball-image {
+  z-index: -100;
+  position: fixed;
+  right: 20px;
+  bottom: 120px;
+  opacity: 0.5;
+}
+tr#frameRow {
+  height: 60px;
+}
+section#header {
+  height: 25%;
+  width: 100%;
+  background-color: #F1BE47;
+  position: fixed;
+  top: 0;
+  left: 0;
+  border-style: solid;
+  border-width: 0px 0px 4px 0px;
+  border-color: #BC9437;
+}
+section#footer {
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  height: 15%;
+  width: 100%;
+  background-color: #F1BE47;
+  border-style: solid;
+  border-width: 4px 0px 0px 0px;
+  border-color: #BC9437;
+  display: block;
+}
```

Comment:
There's an opportunity here to think about styles that are shared across similar HTML components. Can you reduce the duplicate styling across these two different `section` entities?

Reply:
(Think of it like inheritance)